content-service/src/main/java/io/meeds/news/service/impl/NewsServiceImpl.java

```diff
@@ -196,6 +197,9 @@ public class NewsServiceImpl implements NewsService {
 
   @Autowired
   private NewsSearchConnector newsSearchConnector;
+    @Qualifier("newsService")
+    @Autowired
+    private NewsService newsService;
```

Comment:
indentation

Reply:
unnecessary injection of newsService removed !